lib/galaxy/objectstore/s3.py

```diff
@@ -385,9 +385,8 @@ def _get_transfer_progress(self):
     def _get_size_in_s3(self, rel_path):
         try:
             key = self._bucket.get_key(rel_path)
-            if key:
-                return key.size
-        except S3ResponseError:
+            return key.size
+        except Exception:
```

Comment:
If it's not one of the expected errors it should probably not be caught ?

Reply:
This was to have the same logging and return value when `key` is `None`. I can change it to:

```suggestion
        except (S3ResponseError, AttributeError):
```

if that works for you?

Reply:
Rebased with the suggested change.